### PR TITLE
Fix config file locations and clean up test warnings

### PR DESCRIPTION
**Pull request by brienw**
This fixes the issue that @pmeinhardt noted after he reworked/fixed the `IO` verbosity

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
-import_config "#{Mix.env}.exs"
+import_config "deploy/#{Mix.env}.exs"
```
Comment (pmeinhardt) [inline config/config.exs]: Shouldn't this line just be removed? At least it seems a bit confusing to import "deploy" config into the Bootleg config 🤔

Comment (rjanja) [reply config/config.exs]: Yes, good catch @pmeinhardt 👍

**Comment by brienw** [comment]
Docker is giving me the business right now when i'm running the tests to verify that the cleanup of the config is kosher, hopefully resolved soon.

@pmeinhardt  is correct. it turns out this appears to be just cruft from the very early version of bootleg. deploy config files no longer need to be explicitly imported, as they are loaded automatically now when `Bootleg.Config.Agent` is started

**Comment by brienw** [comment]
Ok, all ready @rjanja

**Pull request merged**